docker-compose.yaml

```diff
@@ -100,11 +103,9 @@ services:
       - ./scripts/grafana/provisioning:/etc/grafana/provisioning
       - ./examples:/var/lib/examples
       - /var/lib/grafana/dashboards/
-    entrypoint: ["bash"]
+    entrypoint: [ "bash" ]
     command: > # This is to set the datasource id in the examples to the datasource created for local grafana
-      -cx 'cp -r /var/lib/examples/* /var/lib/grafana/dashboards/
-      && find /var/lib/grafana/dashboards/ -name ''*.json'' | xargs grep -rnl "$${DS_POSTGRES}" | xargs -i@ sed -i "s/\$${DS_POSTGRES}/SXQZgpP7z/g" @
-      && exec grafana-server --config="/etc/grafana/grafana.ini"'
+      -cx 'cp -r /var/lib/examples/* /var/lib/grafana/dashboards/ && find /var/lib/grafana/dashboards/ -name ''*.json'' | xargs grep -rnl "$${DS_POSTGRES}" | xargs -i@ sed -i "s/\$${DS_POSTGRES}/SXQZgpP7z/g" @ && exec grafana-server --config="/etc/grafana/grafana.ini"'
```

Comment:
@ramiroJCB @amenowanna just wanna make sure the changes here are intentional - not super familiar with this part of the `docker-compose`

Reply:
I'm pretty sure than this is a forma issue .re-formatting it again